recipes/flask-oidc/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "flask-oidc" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "0c12151139d47a562e1c5ae203fb9dbc759fe7474cc01e0238bef828ece58f4e"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - oidc-register=flask_oidc.registration_util:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - flask
+    - pip
+    - python
+  run:
+    - flask
+    - itsdangerous
+    - oauth2client
+    - python
+    - six
+
+test:
+  imports:
+    - flask_oidc
+  commands:
+    - oidc-register --help
+  requires:
+    - mock
+    - nose
+
+about:
+  home: "https://github.com/puiterwijk/flask-oidc"
+  license: "BSD"
```

Which BSD licence is it?
```suggestion
  license: BSD-N-Clause
```
(Replace N as needed)